bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,46 @@
+import aiohttp
+import discord
+from discord.ext import commands
+
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.current_channel = None
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with aiohttp.ClientSession() as session:
```

Comment:
Use `self.bot.http_session` here instead of making a new client session

Reply:
oh, okay!